qdrant-landing/content/articles/qdrant-1.9.x.md

```diff
@@ -0,0 +1,88 @@
+
+> **Documentation:** [Read the access level breakdown](/documentation/guides/security/#table-of-access) to see which actions are allowed or denied.
+
+We highly recommend this for enterprises using [Qdrant Hybrid Cloud](https://hybrid-cloud.qdrant.tech/), as it is tailored to those who need additional control over company data and user access. RBAC empowers administrators to define roles and assign specific privileges to users based on their roles within the organization. In combination with [Hybrid Cloud's data sovereign architecture](/documentation/hybrid-cloud/), this feature reinforces internal security and efficient collaboration by granting access only to relevant resources.
+
+**Dashboard users:** For your convenience, we have added a JWT generation tool the Qdrant Web UI under the 🔑 tab. If you're using the default url, you will find it at `http://localhost:6333/dashboard#/jwt`.
+
+## Faster shard transfers on node recovery
+
+We now offer a streamlined approach to [data synchronization between shards](/documentation/guides/distributed_deployment/#shard-transfer-method) during node upgrades or recovery processes. Traditional methods used to transfer the entire dataset, but our new `wal_delta` method focuses solely on transmitting the difference between two existing shards. By leveraging the Write-Ahead Log (WAL) of both shards, this method selectively transmits missed operations to the target shard, ensuring data consistency. 
+
+In some cases, where transfers can take hours, this update **reduces transfers down to a few minutes.**
+
+The advantages of this approach are twofold: 
+1. **It is faster** since only the differential data is transmitted, avoiding the transfer of redundant information. 
+2. It upholds robust **ordering guarantees**, crucial for applications reliant on strict sequencing. 
+
+For more details on how this works, check out the [shard transfer documentation](/documentation/guides/distributed_deployment/#shard-transfer-method).
+
+> **Note:** There are limitations to consider. First, this method only works with existing shards. Second, while the WALs typically retain recent operations, their capacity is finite, potentially impeding the transfer process if exceeded. Nevertheless, for scenarios like rapid node restarts or upgrades, where the WAL content remains manageable, WAL delta transfer is an efficient solution.
+
+Overall, this is a great optional optimization measure and serves as the **auto-recovery default for shard transfers**. By minimizing data redundancy and expediting transfer processes, it alleviates the strain on the cluster during recovery phases, enabling faster node catch-up.
```

Comment:
Maybe it's worth adding something like this:

```suggestion
Overall, this is a great optional optimization measure and serves as the **auto-recovery default for shard transfers**. It's safe to use everywhere because it'll automatically fall back to streaming records transfer if no difference can be resolved. By minimizing data redundancy and expediting transfer processes, it alleviates the strain on the cluster during recovery phases, enabling faster node catch-up.
```

Notably, there are situations where a difference cannot be resolved. The simplest example is when transferring a shard that is not on another node yet. In that case it'll just fall back to streaming records transfer.